astropy/cosmology/parameter/_core.py

```diff
@@ -143,14 +143,13 @@ def __post_init__(self) -> None:
         # attribute name on container cosmology class.
         # really set in __set_name__, but if Parameter is not init'ed as a
         # descriptor this ensures that the attributes exist.
```

Comment:
Isn't this comment now outdated?

Reply:
No, Python will call ``__set_name__`` a second time, if the instance is correctly attached as descriptor (i.e. constructed as another class's field, not attached afterwords). This first time is to ensure that all attributes exist, in case the class is not constructed as a descriptor.

Reply:
I'll update the comment to make it more clear.

Reply:
Done.